Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -119,6 +159,23 @@ def bump_formula_pr
 
     backup_file = File.read(formula.path) unless ARGV.dry_run?
 
+    if new_mirror
+      replacement_pairs << [/^( +)(url \"#{new_url}\"\n)/m, "\\1\\2\\1mirror \"#{new_mirror}\"\n"]
+    end
+
+    if forced_version && forced_version != "0"
+      if File.read(formula.path).include?("version \"#{old_formula_version}\"")
+        replacement_pairs << [old_formula_version.to_s, forced_version]
+      else
+        if new_mirror
+          replacement_pairs << [/^( +)(mirror \"#{new_mirror}\"\n)/m, "\\1\\2\\1version \"#{forced_version}\"\n"]
+        else
+          replacement_pairs << [/^( +)(url \"#{new_url}\"\n)/m, "\\1\\2\\1version \"#{forced_version}\"\n"]
+        end
+      end
+    elsif forced_version && forced_version == "0"
+      replacement_pairs << [/^  version \"[a-z\d+\.]+\"\n/m, ""]
```

Comment:
Yeah, I guess that's the issue with using `InReplace`. I can think of two options:

1. Figure out the range of line numbers where the appropriate spec starts and end, and limit the `InReplace` call to those lines (not sure if `InReplace` can do that, but I've done something similar with `sed`).

2. Modify the formula data structure and serialize it to the file. This would probably strip comments and seems like it would be hard to get right.